perceval/backends/_naive_approx.py

```diff
@@ -45,15 +46,15 @@ def name(self) -> str:
         return "NaiveApprox"
 
     def _compute_permanent(self, M):
-        permanent_with_error = xq.estimate_permanent_cx(M, self._gurvits_iterations, 0)
+        permanent_with_error = xq.estimate_permanent_cx(M, self._gurvits_iterations)
```

Comment:
It seems like `M` is being replaced by `m` at the different locations in this file. You missed it here, was this intentional?